frontend/src/components/pages/connect/Overview.tsx

```diff
@@ -89,14 +128,14 @@ class KafkaConnectOverview extends PageComponent {
                             ? tabs[0].content()
                             : tabs[0].content
                     )
-                    : <Tabs tabs={tabs} defaultSelectedTabKey={showPipelines ? 'redpandaConnect' : 'kafkaConnect'} />
+                    : <Tabs tabs={tabs} defaultSelectedTabKey={getDefaultView(this.props.defaultView).initialTab} />
```

Comment:
I'd personally try to have tabs support different URLs for us to be able to link to the pages. I've re-used some pattern in Settings from some other page. Maybe we could apply here as well?

Reply:
could you point me to that logic?

Reply:
one of my principal goal is to share url with tab in mind

Reply:
solved in a DM